runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -185,6 +185,16 @@ struct StackTraceEntry {
 	StackTraceEntry *next;
 };
 
+struct JVMInformationEntry {
```

Comment:
since the result is the same each time, and you need to create one for each chunk. You coould put the struct in jfrstate

```
typedef struct JFRConstantEvents {
	JVMInformationEntry JVMInfoEntry;
	...
} JFRConstantEvents;

typedef struct JFRState {
	char *jfrFileName;
	U_8 *metaDataBlobFile;
	UDATA metaDataBlobFileSize;
	IDATA blobFileDescriptor;
	void *jfrWriter;
	UDATA jfrChunkCount;
	JFRConstantEvents constantEvents;
} JFRState;
```